__fixtures__/javascript.js

```diff
@@ -1,3 +1,5 @@
+import { unusedEs6Import } from './stub/unusedEs6Import';
+
```

Comment:
should this be a require since it's js?
```suggestion
const { unusedEs6Import } = require( './stub/unusedEs6Import' );
```

Reply:
To cater for babel-ish javascript. i.e. vip-go-admin-console.

I specifically name it as `unusedEs6Import` because `unused-imports` is usable only with es6 imports.